exercise/static/exercise/results_staff.js

```diff
@@ -0,0 +1,840 @@
+let currentScript = (document.currentScript ?
```

Comment:
Missing the anonymous function scope wrapping!

Reply:
I assume I want to wrap the whole script?

Reply:
Actually I dont want to do that, but i am unsure how I would handle these variables I use in functions.

Reply:
Wrap the whole file and export functions that really need it. You can set variables to global scope with `window.variable = foo`. `window` is the global scope in browsers.

Reply:
I used this to do it. My question is how should I indent the file. Should I have one whole indent for the rest of the file just for the wrap function as I have in the current revision?

Reply:
It's good now